src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix filter applied on constant names
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
```

Comment:
from a debug pov... `string("E_WARNING | E_NOTICE")` is the actual representation, whether someone uses `E_WARNING` or `E_SOMETHING_UNRELATED`.. we dont care right? As both are `int(2)`, which in this context we should interpret as `string("E_WARNING")`.. no?